### DISCUSSION TITLE
Earth gone missing

### DISCUSSION THREAD

**Discussion by Hassieflassy** [endless-sky/endless-sky#9632] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

At some point during my playthrough Earth went missing from Sol.

I already checked the game files to see if they got corrupted or something, but both in 'map system.txt' and 'map planets.txt' Earth is present as an object and all the relative information is there.

I don't know exactly when this happened, I searched through my savefile if anything were to point to it, but alas.

Now, I'm stuck in a mission which wants me to go to Earth, but seeing as it's either gone or invisible, I can't finish it.

It doesn't matter if I load up a different save, or start a new game completely, or even reinstall the game. Earth is simply not there.

### Steps to Reproduce

Beats me, honestly

### Expected Behavior

Well, Earth still being there

### Screenshots

![20231227192300_1](https://github.com/endless-sky/endless-sky/assets/155012615/2b5d0177-055d-4f0d-bf04-25d65b788127)

### Operating System

Windows 10 Home 22H2 19045.3448

### Game Source

Steam

### Game Version

v0.10.4

### Additional Information

I'm running the following plugins:

    Aben
    Exporianes
    The Adamas Project
    The-Enclave-0.0.11
    UrsaPolaris
    Endless Sky high DPI

**Comment by lumbar527** [comment]
Do you have any plugins?

**Comment by Hassieflassy** [comment]
Yes, I'm running the following plugins:
- Aben
- Exporianes
- The Adamas Project
- The-Enclave-0.0.11
- UrsaPolaris
- Endless Sky high DPI

**Comment by Hassieflassy** [comment]
So when I enter Sol, there's this target around what should be Earth.

![20231227200002_1](https://github.com/endless-sky/endless-sky/assets/155012615/d1bc6d0c-5f80-41f2-9bbf-01a67b744ec5)

There are even ships taking off from there, but I can't land on it.

**Comment by MCOfficer** [comment]
Some of these are pretty old, so i wouldn't be surprised if they are at fault. Can you send your save file? You can find it in `%APPDATA%\endless-sky\saves`

Edit: It might also be useful to know where you got UrsaPolaris, The Enclave and Adamas from. Afaik multiple versions exists.

**Comment by Hassieflassy** [comment]
Got the plugins from their respective GitHub, so I'm assuming the're up to date?

[Nardix Johnson.txt](https://github.com/endless-sky/endless-sky/files/13780952/Nardix.Johnson.txt)
(Yes, I'm very much aware this save has been edited _alot_. It's my go to 'F*** around and find out' save)

**Comment by MCOfficer** [comment]
> Got the plugins from their respective GitHub, so I'm assuming the're up to date?


Some of them aren't, the best source of old plugins like that is probably [Hecter's archive](https://github.com/Hecter94/EndlessSky-PluginArchive).
 
 ---
 
 
At L23054, there's this:

```
		object Earth
			sprite planet/earthse
			distance 965.669
			period 365.25
			object Luna
				sprite planet/luna
				distance 201.24
				period 27.3
```

That's part of EnigmaCorp, so my best guess is that you had EnigmaCorp installed at some point, and it changed earth's sprite to `planet/earthse`. Now that you uninstalled it, there is no valid sprite for earth, so nothing is being shown in sol.

You *should* be able to fix it by removing the entire `system Sol` (L23042) from your savefile. Either way, i'm reasonably sure this is not an issue with vanilla ES *[edit:] , so let's move troubleshooting to a discussion.*

**Comment by Hassieflassy** [comment]
Thank you, that did the trick!
Even removing the "se" part already worked!
